docs/concepts/sides.md

```diff
@@ -34,10 +34,12 @@ How do we resolve this? Luckily, FML provides us with a `@SidedProxy` annotation
 
 !!! note
 
-    It is important to understand that FML picks the proxy to instantiate based on the physical side. A single player world (logical server, logical client within a physical client) will still have a proxy of the type you specify in `clientSide`! If you want behaviours that should run on both sides, make your client proxy extend from the server proxy and call the super class method. Make sure your server proxy does not reference any classes that only exist on the physical server.
+    It is important to understand that FML picks the proxy to instantiate based on the physical side. A single player world (logical server, logical client within a physical client) will still have a proxy of the type you specify in `clientSide`! If you want behaviours that should run on both physical sides, you must arrange for that to happen outside of the proxies
```

Comment:
I don't see why this bit is needed at all. It's not related to the integrated vs dedicated note. IMO it should talk about how to run code based on the logical side, using the aforementioned `isRemote`.